tests/components/airvisual/test_config_flow.py

```diff
@@ -95,10 +148,11 @@
         )
 
         assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
-        assert result["title"] == "Cloud API (API key: abcd...)"
+        assert result["title"] == "Cloud API (51.528308, -0.3817765)"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5238698)